docs/maps/import-geospatial-data.asciidoc

```diff
@@ -0,0 +1,46 @@
+[role="xpack"]
+[[import-geospatial-data]]
+== Import geospatial data
+
+To import geospatical data into the Elastic Stack, the data must be indexed as {ref}/geo-point.html[geo_point] or {ref}/geo-shape.html[geo_shape].
+Geospatial data comes in many formats.
+Choose an import tool based on the format of your geospatial data.
+
+[discrete]
+=== Upload CSV with latitude and longitude columns
+
+*File Data Visualizer* indexes CSV files with latitude and longitude columns as a geo_point.
+
+. Click *{kib} > Machine Learning* from the side navigation.
```

Comment:
```suggestion
. Open the menu, and click *Machine Learning*.
```

Reply:
How about changing to "Open the side navigation, and click Machine Learning". "Open the menu" is too vague. There are a log to menus in Kibana and just saying "Open the menu" does not properly direct the user on where to go

Reply:
OK